dlt/common/libs/pyarrow.py

```diff
@@ -530,22 +514,57 @@ def normalize_py_arrow_item(
         new_fields.append(schema.field(idx).with_name(column_name))
         new_columns.append(item.column(idx))
 
-    if needs_load_id and load_id:
-        # Storage efficient type for a column with constant value
-        load_id_type = pyarrow.dictionary(pyarrow.int8(), pyarrow.string())
-        new_fields.append(
-            pyarrow.field(
-                naming.normalize_identifier(C_DLT_LOAD_ID),
-                load_id_type,
-                nullable=False,
-            )
-        )
-        new_columns.append(pyarrow.array([load_id] * item.num_rows, type=load_id_type))
-
     # create desired type
     return item.__class__.from_arrays(new_columns, schema=pyarrow.schema(new_fields))
 
 
+def add_dlt_load_id_column(
+    item: TAnyArrowItem,
+    columns: TTableSchemaColumns,
+    caps: DestinationCapabilitiesContext,
+    naming: NamingConvention,
+    add_load_id: bool,
```

Comment:
why do you need this? this just skips the whole code inside. so just don't call this function instead